### PR TITLE
Clean up symbol upload script logs

### PR DESCRIPTION
**Pull request by tustanivsky**
Related to #636 

#skip-changelog

### CODE REVIEW

File: plugin-dev/Scripts/upload-debug-symbols-win.bat [hunk 1]
```diff
@@ -33,11 +33,10 @@ if "%TargetPlatform%"=="Win64" (
 )
 
 call :ParseIniFile "%ConfigPath%\DefaultEngine.ini" /Script/Sentry.SentrySettings UploadSymbolsAutomatically UploadSymbols
-echo Sentry: Automatic symbols upload is '%UploadSymbols%' from plugin settings
```
Comment (bitsandfoxes) [inline plugin-dev/Scripts/upload-debug-symbols-win.bat:36]: Won't we miss that log?

File: plugin-dev/Scripts/upload-debug-symbols.sh [hunk 2]
```diff
@@ -31,11 +31,10 @@ else
 fi
 
 UPLOAD_SYMBOLS=$(awk -F "=" '/UploadSymbolsAutomatically/ {print $2}' "${CONFIG_PATH}/DefaultEngine.ini")
-echo "Sentry: Automatic symbols upload is '$UPLOAD_SYMBOLS' from plugin settings"
```
Comment (bitsandfoxes) [inline plugin-dev/Scripts/upload-debug-symbols.sh:34]: Won't we miss that log?

**Review by bitsandfoxes — COMMENTED** [review]
Improved logs! <3

File: plugin-dev/Scripts/upload-debug-symbols-win.bat (hunk 1 above)
Comment (tustanivsky) [reply plugin-dev/Scripts/upload-debug-symbols-win.bat:36]: If symbol upload flag is unspecified/disabled in settings or overridden by env variable we'll print a corresponding message later on anyway, so keeping this message here doesn't seem to be necessary any longer.

**Review by bitsandfoxes — APPROVED** [review]

**Pull request merged**